### ISSUE TITLE
Automatic backup plugin "dbInclude" does not appear to work

### ISSUE THREAD

**Issue by obi1kenobi** [orientechnologies/orientdb#6662]
## OrientDB Version, operating system, or hardware.
- v2.0 SNAPSHOT[ ] - .18[ ] .17[ ] .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.1 SNAPSHOT[ ] - .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.2 SNAPSHOT[ ] - .8 [x] .rc1[ ] .beta2[ ] .beta1[ ]

## Operating System
- [x] Linux
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

## Expected behavior and actual behavior

In 2.1.x, the OAutomaticBackup plugin took XML configuration via the `orientdb-server-config.xml` file, and supported the following parameter:
```
<parameter name="db.include" value="my_database"/>
```

Per the (newest docs)[https://orientdb.com/docs/last/Automatic-Backup.html], in 2.2.8 the preferred way to specify backup properties is via the `automatic-backup.json` file, and the equivalent parameter is named `dbInclude`. I added the following data to my backup JSON file, which is otherwise identical to the default file:
```
  "dbInclude": [
    "my_database"
  ],
```

However, no backups happen, and I see lines like the following in the logs:
```
2016-09-06_00:00:00.01039 2016-09-06 00:00:00:010 INFO  Automatic Backup finished: 0 ok, 0 errors [OAutomaticBackup$1]
```

Removing the `dbInclude` key results in correct backup of all databases.

Note that I have not changed the database name between 2.1.x and 2.2.8, and I have also ensured that `db.include` and `dbInclude` in the XML and JSON files respectively use the exact same name. This suggests that a silent breaking change has occurred in the configuration of the backup plugin between 2.1.x and 2.2.8.

## Steps to reproduce the problem

Create a new database named `my_database`. Add the following to the default JSON backup file:
```
  "dbInclude": [
    "my_database"
  ],
``` 
Enable automatic backups in the config XML file, pointing to the JSON config file for configuration. Observe as your database is not backed up, unlike in 2.1.x.

**Comment by maggiolo00** [comment]
hi @obi1kenobi 

i think the correct way of databases list is like the old one with comma separated values

```
dbInclude": "my_database",
```

**Comment by obi1kenobi** [comment]
From  the documentation: https://orientdb.com/docs/last/Automatic-Backup.html

```
- "dbInclude" Defines in a list the databases to include in the automatic backups. If empty, it backs up all databases.
```

The file is JSON, where a list is a supported data structure that isn't the same thing as a comma-separated string. I think the following things are necessary:
- The documentation needs some rewording and probably an example.
- If you are set on using comma-separated strings when lists are available, then I think OrientDB should throw an error when an unexpected type is encountered. Imagine how bad of a surprise it would be for your database to crash, and for you to only then discover that backups weren't turned on because you provided a list and the database expected a comma-separated string.

**Comment by maggiolo00** [comment]
@obi1kenobi 

i guess is that for backward compatibility since the configuration was String in ODB < 2.2 

The JSON rep should support [] notation.

**Comment by maggiolo00** [comment]
@obi1kenobi  Fixed

**Comment by obi1kenobi** [comment]
Thanks! I assume this'll be released in 2.2.10?

**Comment by maggiolo00** [comment]
@obi1kenobi 

yes 2.2.10
